### ISSUE TITLE
Temporary Tables

### ISSUE THREAD

**Issue by imliam** [calebporzio/sushi#63]
Using a separate SQLite connection from the main database poses some problems in practice, such as not being able to use any joins or relationships that rely on joins. It also makes behaviour between models inconsistent, as there are various quirks with how data is returned/processed in MySQL/Postgres/SQLite/etc. so mixing two of them can lead to odd situations.

Instead of relying on a different connection entirely, how feasible would it be to make Sushi instead create a temporary table on the default connection? This would make all behaviour consistent, allow for direct SQL queries with driver-specific language, allow joins, and remove the reliance on SQLite entirely.

**Comment by Mushr0000m** [comment]
Yes it would help a lot, in my case I can't use a `belongsToMany` relation because it uses `join`...

**Comment by ryangjchandler** [comment]
@imliam what sort of caching would make the most sense here?
